localstack-core/localstack/services/apigateway/next_gen/execute_api/gateway.py

```diff
@@ -0,0 +1,33 @@
+from rolo import Response
+from rolo.gateway import Gateway
+
+from . import handlers
+from .context import RestApiInvocationContext
+
+
+class RestApiGateway(Gateway):
```

Comment:
nit: Would be great to add a bit of documentation here: This is the main part controlling the API GW invocation handling now. Maybe we can add a few sentences (can also be in upcoming iterations) on what this is and how to extend / plug into this.